alpine/sql-requirements.in

```
@@ -1,7 +1,13 @@
--r wsgi-requirements.txt  # Maintain dependencies of the base image.
+# Maintain exact dependencies of the base image.
+-r base-requirements.txt
```

I'm wondering if the absolute path should be preferred? That way the pip-sync can be executed from anywhere in child images without problems.

```suggestion
-r /opt/base-requirements.txt
```